src/components/Sidebar/Sidebar.tsx

```diff
@@ -23,6 +24,11 @@ const Sidebar = (): JSX.Element => {
   const width = 250;
   const [xPosition, setX] = useState(-width);
   const ref = React.useRef(null);
+  const { list } = useContext(CartContext);
+  let qty = 0;
```

Comment:
remove this and the below calculis in favour of https://github.com/mrplum/plum_cart/pull/17/files#r522331580